ohsomeTools/__init__.py

```diff
@@ -29,7 +29,7 @@
 from datetime import datetime
 
 
-# noinspection PyPep8Naming
+# noinspection PyPep8Namingcis
```

Comment:
What does the `cis` part do?

Reply:
I'm not sure. I don't remember putting this here, and it's not even in my local code.

Reply:
We'll leave it as is for now.